pull_data.php

```diff
@@ -1,11 +1,9 @@
 <?php
 
-
 $db = new PDO('mysql:host=127.0.0.1;dbname=portfolio', 'root');
 
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,PDO::FETCH_ASSOC);
 
-
 function about_output(pdo $db) :array
 {
```

Comment:
Take functions outside of the place in which they are used. Preferably in a new file with a similar filename eg filename_function.php.